src/components/PostForm.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  Stack,
+  StackDivider,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { BlogPostCategory } from "../types/blogPost";
+
+export interface PostFormType {
```

Comment:
The naming convention is to use `Props` suffix for the props. So the interface name should be `PostFormProps`